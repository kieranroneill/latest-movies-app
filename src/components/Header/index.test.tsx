import { shallow } from 'enzyme';
import * as React from 'react';

// Components.
import {
    Header,
    Props
} from './';

describe('src/components/Header', () => {
    describe('<Header /> snapshots', () => {
        it('should match the snapshot with default props', () => {
            const props: Props = {
                layout: {
                    page: {
                        title: 'Some catchy title'
                    }
                }
            };

            expect(
                shallow(
                    <Header {...props} />
                )
            ).toMatchSnapshot();
        });
    });
});
