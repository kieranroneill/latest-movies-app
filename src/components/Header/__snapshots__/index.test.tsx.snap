// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/components/Header <Header /> snapshots should match the snapshot with default props 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header
    layout={
      Object {
        "page": Object {
          "title": "Some catchy title",
        },
      }
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": <WithStyles(Toolbar)>
        <WithStyles(Typography)
          color="inherit"
          variant="title"
        >
          Some catchy title
        </WithStyles(Typography)>
      </WithStyles(Toolbar)>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <WithStyles(Typography)
          color="inherit"
          variant="title"
        >
          Some catchy title
        </WithStyles(Typography)>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": "Some catchy title",
          "color": "inherit",
          "variant": "title",
        },
        "ref": null,
        "rendered": "Some catchy title",
        "type": [Function],
      },
      "type": [Function],
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <WithStyles(Toolbar)>
          <WithStyles(Typography)
            color="inherit"
            variant="title"
          >
            Some catchy title
          </WithStyles(Typography)>
        </WithStyles(Toolbar)>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": <WithStyles(Typography)
            color="inherit"
            variant="title"
          >
            Some catchy title
          </WithStyles(Typography)>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": "Some catchy title",
            "color": "inherit",
            "variant": "title",
          },
          "ref": null,
          "rendered": "Some catchy title",
          "type": [Function],
        },
        "type": [Function],
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
